proof_range.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/tendermint/tendermint/crypto/tmhash"
-	cmn "github.com/tendermint/tendermint/libs/common"
+	cmn "github.com/tendermint/tmlibs/common"
```

Comment:
Is there a reason this doesn't use `github.com/tendermint/tendermint/libs/common`? I mean, if we have to depend on tendermint, than let's use it.

Reply:
fixed on another PR i'll make.